caPutLogApp/caPutLog.c

```diff
@@ -37,6 +37,13 @@
 #define LOCAL static
 #endif
 
+int caPutLogInitialized = 0;
+
+int caPutLogWasInit()
```

Comment:
Nitpicky, but I think that the function name being `caPutLogInitialised` (that is, as a verb) would be nicer, and the variable as something like `_caPutLogInitialised` (i.e. an "internal" variable name).